airbyte-integrations/connectors/source-twilio/source_twilio/manifest.yaml

```
@@ -0,0 +1,832 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    extractor:
+      field_path:
+        - "{{ parameters['name'] }}"
+  requester:
+    url_base: "https://api.twilio.com/2010-04-01"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['account_sid'] }}"
+      password: "{{ config['auth_token'] }}"
+  datetime_cursor:
+    type: "DatetimeBasedCursor"
+    start_datetime:
+      datetime: "{{ config['start_date'] }}"
+      datetime_format: "%Y-%m-%d"
+    end_datetime:
+      datetime: "{{ now_utc() }}"
+      datetime_format: "%Y-%m-%d %H:%M:%S.%f+00:00"
+    step: "P1D"
+    datetime_format: "%Y-%m-%d"
+    cursor_granularity: "P1D"
+    lookback_window: "{{ config['lookback_window']}}"
+    cursor_field: "date"
+  retriever:
+    record_selector:
+      $ref: "#/definitions/selector"
+    requester:
+      $ref: "#/definitions/requester"
+  default_paginator:
+    type: "DefaultPaginator"
+    page_size_option:
+      inject_into: "request_parameter"
+      field_name: "PageSize"
+    pagination_strategy:
+      type: "CursorPagination"
+      cursor_value: "{{response['next_page_uri']}}"
+      page_size: 50
+    page_token_option:
+      type: RequestPath
+  base_stream:
+    retriever:
+      $ref: "#/definitions/retriever"
+  base_stream_with_pagination:
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/default_paginator"
+
+  accounts_stream:
+    $ref: "#/definitions/base_stream_with_pagination"
+    $parameters:
+      name: "accounts"
+      path: "/Accounts.json"
+      primary_key: "sid"
+  
+  accounts_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/accounts_stream"
+        parent_key: sid
+        partition_field: id
+
+  addresses_stream:
+    $ref: "#/definitions/base_stream_with_pagination"
+    $parameters:
+      name: "addresses"
+      path: "/Accounts/{{ stream_slice.id }}/Addresses.json"
+      primary_key: "sid"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+      paginator:
+        $ref: "#/definitions/default_paginator"
+      partition_router:
+        $ref: "#/definitions/accounts_partition_router"
```

I think you only need to override the new part
```suggestion
  addresses_stream:
    $ref: "#/definitions/base_stream_with_pagination"
    $parameters:
      name: "addresses"
      path: "/Accounts/{{ stream_slice.id }}/Addresses.json"
      primary_key: "sid"
    retriever:
      partition_router:
        $ref: "#/definitions/accounts_partition_router"
```

---

@marcosmarxm what do you mean by overriding the new part?

---

You don't need to specify all components again, only the ones you want to change.